services/template.py

```diff
@@ -0,0 +1,21 @@
+from jinja2 import Environment, PackageLoader, StrictUndefined, select_autoescape
+
+
+class TemplateService:
+    env = None
+
+    @classmethod
+    def loaded(cls):
+        return cls.env is not None
+
+    def __init__(self):
+        # this loads the templates from the templates directory in this repository since it's looking for a dir named `templates` next to app.py
+        TemplateService.env = Environment(
```

Comment:
this will reinitialize `TemplateService.env` each time we create an instance, we should only initialize it if it's still `None`